R/agglomerate.R

```diff
@@ -1,12 +1,19 @@
-#' Agglomerate data using taxonomic information
+#' Agglomerate or merge data using taxonomic information
 #'
 #' \code{agglomerateByRank} can be used to sum up data based on associations
 #' with certain taxonomic ranks, as defined in \code{rowData}. Only available
-#' \code{\link{taxonomyRanks}} can be used.
+#' \code{\link{taxonomyRanks}} can be used. 
+#' 
+#' \code{agglomerateByVariable} merges data on rows or columns of a 
+#' \code{SummarizedExperiment} as defined by a \code{factor} alongside the 
+#' chosen dimension. Metadata from the \code{rowData} or \code{colData} are 
+#' retained as defined by \code{archetype}.
+#' \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}} are 
+#' agglomerated, i.e. summed up. If the assay contains values other than counts 
+#' or absolute values, this can lead to meaningless values being produced. 
```

Comment:
User wonders what is the difference between *Rank and *Variable.

*Rank: the data is agglomerated by taking into account the whole taxonomy table. --> use if you want to agglomerate based on taxonomy rank.
*Variable: the data is agglomerated based on certain variable. --> use if you want to agglomerate based on some other variable

Try to rephrase that